### PR TITLE
Disable analitics

### LINKED ISSUE #769: Add ability to disable any analytics
Add an option to settings

- Crashlytics
- Branch
- Answer

### PR DESCRIPTION
**Pull request by OlegGordiichuk**
- Add new settings button
- Simple disable of the Branch and Crashlytics
- Complex disable of the Answer (with alert and **restart** of the app option)

Fix #769

### CODE REVIEW

File: Trust/Settings/Controllers/AnaliticsViewController.swift [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+            <<< SwitchRow {
+                $0.title = viewModel.crashlytics.name
+                $0.value = viewModel.crashlytics.isEnabled
+            }.onChange { [weak self] row in
+                 self?.viewModel.crashlytics.update(with: row.value ?? false)
+            }
+        }
+
+    private func showAnswerAlert() {
```
Comment (vikmeup) [inline Trust/Settings/Controllers/AnaliticsViewController.swift]: can you remove this? It's ok if it's gonna disable crashes after the restart.

Comment (OlegGordiichuk) [reply Trust/Settings/Controllers/AnaliticsViewController.swift]: I left only notification for the client to restart app.

Comment (vikmeup) [reply Trust/Settings/Controllers/AnaliticsViewController.swift]: No need, it's an extra code.

**Comment by OlegGordiichuk** [comment]
Done

File: Trust/Settings/Types/Analitics.swift [hunk 2]
```diff
@@ -0,0 +1,31 @@
+// Copyright SIX DAY LLC. All rights reserved.
+
+import UIKit
+
+enum Analitics: String {
+    case crashlytics
+    case branch
+    case answer
+
+    var name: String {
```
Comment (vikmeup) [inline Trust/Settings/Types/Analitics.swift]: what does this mean to the user?

File: Trust/Settings/Types/Analitics.swift [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        switch self {
+        case .crashlytics: return "Crashlytics"
+        case .branch: return "Branch"
+        case .answer: return "Answer"
+        }
+    }
+
+    var isEnabled: Bool {
```
Comment (vikmeup) [inline Trust/Settings/Types/Analitics.swift]: use `PreferencesController`

File: Trust/Settings/Types/Analitics.swift (hunk 2 above)
Comment (OlegGordiichuk) [reply Trust/Settings/Types/Analitics.swift]: Done

File: Trust/Settings/Types/Analitics.swift (hunk 3 above)
Comment (OlegGordiichuk) [reply Trust/Settings/Types/Analitics.swift]: Done

**Comment by OlegGordiichuk** [comment]
@vikmeup I add description.

simulator screen shot - iphone x - 2018-06-27 at 14 06 59

**Pull request merged**